js/generate.js

```diff
@@ -38,23 +38,24 @@ function fillCell(el, data, image) {
 }
 
 function importSVG(svgurl, el, postHook) {
-    var xhr = new XMLHttpRequest();
-    xhr.open("GET",svgurl);
-    xhr.responseType = "document";
-    xhr.onload = function (e) {
-	if (e.target.status == "200" || e.target.status == "304") {
-	    var svg = e.target.response.documentElement;
-	    svg.querySelector("style").remove();
-	    el.appendChild(svg);
-	    if (postHook) {
-		postHook(el);
-	    }
-	}
-    };
-    xhr.send();
+    fetch(svgurl, {mode:'cors'}).then(response => response.text())
```

Comment:
So, worried that this will break iOS and MS Edge, etc. that don't yet support fetch().